courses/free-courses-en.md

```diff
@@ -1186,6 +1186,12 @@
 * [Functional Programming in Scala Specialization](https://www.coursera.org/course/reactive)
 
 
+### Serverless
+
+* [Serverless framework getting started](https://www.serverless.com/framework/docs/getting-started)
```

Comment:
Looks like documentation and there is no learning path or connection between the different sections. Is this a value add?

Reply:
We'll accept this as a book, but not as its own category. It's a node framework?